CriticalMass/CMLogoActivityView.swift

```diff
@@ -0,0 +1,426 @@
+//
+//  CriticalMapsActivityView.swift
+//  CriticalMaps
+//
+//  Created by Malte Bünz on 20.07.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+@IBDesignable public class CMLogoActivityView: UIView, IBConstructable {
```

Comment:
Even after setting the FPS target to 60, the first few frames can't keep that Framerate and drop to 50 FPS. Maybe replacing the DrawRect with two CALayer (foreground and background) and only rotate the layer via CoreAnimation might help

Reply:
I was trying to do that after the logo redesign but with the paths from PaintCode it is difficult to find a satisfying solution.

Reply:
I will try to work on this a little to make it more performant